content/ja/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -241,4 +241,4 @@ spec:
     app: iis-2019
 ```
 
-[RuntimeClass]: https://kubernetes.io/docs/concepts/containers/runtime-class/
+[RuntimeClass]: https://kubernetes.io/ja/docs/concepts/containers/runtime-class/
```

Comment:
It is right that there is a Japanese link, but should this be defined as a relative path?

Reply:
depends on base(en) document